src/components/DepositWidget/Styled.tsx

```diff
@@ -84,6 +84,10 @@ export const TokenRow = styled.tr`
     margin: 0;
     padding: 0;
 
+    li {
+      white-space: nowrap;
```

Comment:
careful with this, if the WETH/ETH amounts are large this may cause formatting issues. But maybe we can just `formatSmart` with this later on

Reply:
That's was my thinking
Now we are sure of the currency here. We won't see huge values. These are the top ethereum accounts.
I don't see this accounts of Huobi, Binance as a user here, and with the smart format even would show sth decent